types/gestalt/index.d.ts

```diff
@@ -415,24 +415,28 @@ export interface ComboBoxProps {
     label: string;
     options: ComboBoxItemType[];
     noResultText: string;
-    disabled?: boolean;
-    errorMessage?: string;
-    helperText?: string;
-    inputValue?: string;
-    labelDisplay?: 'visible' | 'hidden';
-    onChange?: (args: { event: React.SyntheticEvent<HTMLInputElement>; value: string }) => void;
-    onBlur?: (args: { event: React.FocusEvent<HTMLInputElement> | React.SyntheticEvent<HTMLInputElement>; value: string }) => void;
-    onFocus?: (args: { event: React.FocusEvent<HTMLInputElement>; value: string }) => void;
-    onKeyDown?: (args: { event: React.KeyboardEvent<HTMLInputElement>; value: string }) => void;
-    onClear?: () => void;
+    zIndex?: Indexable | undefined;
+    disabled?: boolean | undefined;
+    errorMessage?: string | undefined;
+    helperText?: string | undefined;
+    inputValue?: string | undefined;
+    labelDisplay?: 'visible' | 'hidden' | undefined;
+    onChange?: (args: { event: React.SyntheticEvent<HTMLInputElement>; value: string }) => void | undefined;
```

Comment:
did you mean for the type of those methods e.g. `onChange` to be `((args: { event: React.SyntheticEvent<HTMLInputElement>; value: string }) => void) | undefined`? the way they are written means they can *return* `undefined`, not that the methods themselves can *be* `undefined`.

Reply:
Yeah, I'm missig brackets [😅](https://emojipedia.org/grinning-face-with-sweat/)